### DISCUSSION TITLE
[Feature Request] add a search bar within a feed

### DISCUSSION THREAD

**Discussion by yvesPresent** [spacecowboy/Feeder#162] [Ideas]
Hello,
Is it possible to add a search bar to filter the articles by title in memory on the fly ideally.

When i search for items in a field i'd like non matching items to be hidden until i empty the search field via a button. Ideally articles should still be openable via a click on their title (they should work as they currently do).

**Comment by navilg** [comment]
A search box would be really great feature to search a text or pattern from feeds.
@spacecowboy

**Comment by tejonaco** [comment]
I would like it to, even on "All feeds" page (if it's not too much)

**Comment by pirofti** [comment]
First, thank you very much for this great software! 

I kept looking for it and could not find any search functionality. Came here to see if other people have the same issues :)

This is very important to me for keywords searching in large arXiv daily feeds such as cs.LG for example.

**Comment by bakinazik** [comment]
Search bar needed!

**Comment by pirofti** [comment]
I think this was fixed in #647 and can be closed now.
